content/en/docs/Reference/declarative-config.md

```diff
@@ -0,0 +1,341 @@
+---
+title: "Declarative Config"
+linkTitle: "Declarative Config"
+weight: 4
+date: 2021-07-29
+---
+
+Declarative Config (DC) is the latest iteration of OLM's index format. It is a fully plaintext-based (JSON or YAML) evolution of the previous sqlite database format that is fully backwards compatible.
+
+## Motivation
+The primary motivations for this new format are to enable index editting, composability, and extensibility. 
+
+### Editting
+
+With DC, users interacting with the contents of an index are able to make direct changes to the index format and verify that their changes are valid. 
+
+Because the index is now stored in plaintext rather than an sqlite database, index maintainers can update an index without the use of a custom index-aware tool, like `opm`.
+
+This editability opens the door for new features and extensions that were otherwise difficult to implement and maintain in a single tool. For example:
+- Promoting an existing bundle to a new channel
+- Changing the default channel of a package
+- Adding, updating, or removing upgrade edges
+
+### Composability
+
+Declarative Configs are stored in an arbitrary directory hierarchy, which enables index composition. If I have two separate DC directories, `indexA` and `indexB`, I can make a new combined index by making a new directory `indexC` and copying `indexA` and `indexB` into it.
+
+This composability enables decentralized indexes. The format permits operator authors to maintain operator-specific indexes and index maintainers to trivially build an index composed of individual operator indexes.
```

Comment:
question, with a decentralized index approach, how is conflict resolution handled?  For example, 2 different authors use the same metadata/names by mistake or think they own a particular name, I guess one would take precedent over the other when the final index is built?  One would be kicked out in error?

Reply:
Duplicate package names and bundle names within a package would result in an error that's caught by `opm validate`.